src/algorithm/MEMCPY-Cuda.cpp

```diff
@@ -105,7 +107,7 @@ void MEMCPY::runCudaVariantBlock(VariantID vid)
     for (RepIndex_type irep = 0; irep < run_reps; ++irep) {
 
       const size_t grid_size = RAJA_DIVIDE_CEILING_INT(iend, block_size);
-      memcpy<block_size><<<grid_size, block_size>>>(
+      memcpy<block_size><<<grid_size, block_size, 0, res.get_stream()>>>(
```

Comment:
```suggestion
      constexpr size_t shmem_sz = 0;
      memcpy<block_size><<<grid_size, block_size, 0, res.get_stream()>>>(
```

for clarity?

Reply:
That's probably best practice, I can put that in if people would like that to be made more clear.

Reply:
I agree.

Reply:
done